src/hub.js

```diff
@@ -243,26 +240,33 @@ const onReady = async () => {
 
   registerNetworkSchemas();
 
-  const uiRoot = mountUI(scene);
+  mountUI(scene);
+
+  const remountUI = () => {
```

Comment:
maybe set things up so instead of having to mutate `uiRootProps` everywhere, the k/vs to alter are arguments to this function?